### PR TITLE
Feature/cases order & interface translations

### PR DESCRIPTION
**Pull request by ru-danko**

### CODE REVIEW

File: src/translations/data.json [hunk 1]
```diff
@@ -591,8 +591,8 @@
...
     "be": "",
     "pl": "Używamy plików cookie i innych technologii śledzenia, aby poprawić jakość przeglądania naszej witryny, wyświetlać spersonalizowane treści i reklamy, analizować ruch w naszej witrynie i wiedzieć, skąd pochodzą nasi użytkownicy. Przeglądając naszą stronę, wyrażasz zgodę na używanie przez nas plików cookie i innych technologii śledzenia.",
-    "cs": "",
-    "sk": "",
+    "cs": "Používáme soubory cookie a další technologie sledování, abychom zlepšili vaše prohlížení našich webových stránek, zobrazovali vám přizpůsobený obsah a cílené reklamy, analyzovali provoz na našich webových stránkách a porozuměli, odkud naši návštěvníci přicházejí. Procházením našich webových stránek souhlasíte s naším používáním souborů cookie a dalších sledovacích technologií.",
+    "sk": "Používame súbory cookie a ďalšie technológie sledovania, aby sme zlepšili vaše prezeranie našich webových stránok, zobrazovali vám prispôsobený obsah a cielené reklamy, analyzovali prevádzku na našich webových stránkach a porozumeli, odkiaľ naši návštevníci prichádzajú. Prechádzaním našich webových stránok súhlasíte s naším používaním súborov cookie a ďalších sledovacích technológií.",
```
Comment (matobet) [inline src/translations/data.json:595]: it's probably more suitable "prehliadanie"
```suggestion
    "sk": "Používame súbory cookie a ďalšie technológie sledovania, aby sme zlepšili vaše prehliadanie našich webových stránok, zobrazovali vám prispôsobený obsah a cielené reklamy, analyzovali prevádzku na našich webových stránkach a porozumeli, odkiaľ naši návštevníci prichádzajú. Prehliadaním našich webových stránok súhlasíte s naším používaním súborov cookie a ďalších sledovacích technológií.",
```

File: src/translations/data.json [hunk 2]
```diff
@@ -681,8 +681,8 @@
     "uk": "Використовувати форми словозміни при пошуку міжслав'янською мовою",
     "be": "Выкарыстоўваць формы словазмянення пры пошуку на міжславянскай мове",
     "pl": "Szukaj także odmienionych form międzysłowiańskich słów",
-    "cs": "Používat inflexní formy slov při hledání v mezislovanském jazyce",
-    "sk": "Používať inflexné formy slov pri hľadaní v mezislovanském jazyku",
+    "cs": "Používat ohebné formy slov při hledání v mezislovanském jazyce",
+    "sk": "Používať ohebné formy slov pri hľadaní v mezislovanském jazyku",
```
Comment (matobet) [inline src/translations/data.json:685]: ohEbne is czech, if we want this word it should be "ohybné" 
```suggestion
    "sk": "Používať ohybné formy slov pri hľadaní v mezislovanském jazyku",
```
but I believe better would be "formy skloňovania" or simply just "skloňnovanie"
```suggestion
    "sk": "Používať formy skloňovania slov pri hľadaní v mezislovanském jazyku",
```
```suggestion
    "sk": "Používať skloňovanie slov pri hľadaní v mezislovanském jazyku",
```

File: src/translations/data.json [hunk 3]
```diff
@@ -723,1200 +723,1246 @@
...
   },
+  "particle": {
+    "en": "particle",
+    "isv": "čestica",
+    "ru": "частица",
+    "uk": "частка",
+    "be": "часціца",
+    "pl": "partykuła",
+    "cs": "částice ",
+    "sk": "častica ",
```
Comment (matobet) [inline src/translations/data.json:895]: should cs and sk strings contain the trailing space?

File: src/translations/data.json [hunk 4]
```diff
@@ -723,1200 +723,1246 @@
...
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "субстантивований",
+    "be": "субстантываваны",
+    "pl": "rzeczownikowy",
+    "cs": "substantivná",
+    "sk": "substantívná",
```
Comment (matobet) [inline src/translations/data.json:1210]:
```suggestion
    "sk": "substantívna",
```

File: src/translations/data.json [hunk 5]
```diff
@@ -723,1200 +723,1246 @@
...
+    "pl": "Stopnie porówniania",
+    "cs": "Stupně srovnání",
+    "sk": "Stupne porovnania",
+    "sl": "Stopnje primerjave",
+    "hr": "Stupnjevi usporedbe",
+    "sr": "Степени поређења",
+    "mk": "Степени на споредување",
     "bg": "Степени за сравнение"
   },
   "positive": {
```
Comment (matobet) [inline src/translations/data.json:1623]: in slovak (grammatical context) I encountered mostly the terms "pozitív" "komparatív" and "superlatív" respectively

File: src/translations/data.json [hunk 6]
```diff
@@ -723,1200 +723,1246 @@
...
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "умовний спосіб",
+    "be": "умоўны лад",
+    "pl": "tryb przypuszcz.",
+    "cs": "podmiňovací",
+    "sk": "podmienečný",
```
Comment (matobet) [inline src/translations/data.json:1751]:
```suggestion
    "sk": "podmieňovací",
```
or
```suggestion
    "sk": "podmieňovací sp.",
```

File: src/translations/data.json [hunk 7]
```diff
@@ -723,1200 +723,1246 @@
...
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "неозначена форма",
+    "be": "інфінітыў",
+    "pl": "bezokolicznik",
+    "cs": "neurčitý",
+    "sk": "neurčitý",
```
Comment (matobet) [inline src/translations/data.json:1766]: "neurčitok" ? And in that case maybe also in czech "neurčitek"?

File: src/translations/data.json [hunk 8]
```diff
@@ -2383,5 +2429,20 @@
...
+  },
+  "orderOfCases": {
+    "en": "Order of cases",
+    "isv": "Poredok padežev",
+    "ru": "Порядок падежей",
+    "uk": "Порядок відмінків",
+    "be": "Парадак склонаў",
+    "pl": "Kolejność przypadków",
+    "cs": "Pořadí případu",
+    "sk": "Poradie prípadov",
```
Comment (matobet) [inline src/translations/data.json:2441]:
```suggestion
    "sk": "Poradie pádov",
```
and also in czech most likely "Pořadí pádů"

**Review by matobet — CHANGES_REQUESTED** [review]
Reviewed slovak translations.

File: src/translations/data.json (hunk 5 above)
Comment (ru-danko) [reply src/translations/data.json:1623]: let's change

File: src/translations/data.json (hunk 7 above)
Comment (ru-danko) [reply src/translations/data.json]: sdělano

File: src/translations/data.json (hunk 8 above)
Comment (ru-danko) [reply src/translations/data.json]: Czech phrase is also changed, thanks

File: src/translations/data.json (hunk 2 above)
Comment (ru-danko) [reply src/translations/data.json]: I chose the third option

File: src/translations/data.json (hunk 5 above)
Comment (ru-danko) [reply src/translations/data.json:1623]: I changed

**Review by scherebedov — APPROVED** [review]

**Pull request merged**